containers/kubernetes/reference-content/secure-cluster-with-private-network.mdx

```diff
@@ -77,20 +69,78 @@ What will happen after starting the migration:
+| Notice      | Default choice for new clusters. Can be used in combination with pools using full isolation.                                                    | Requires a Public Gateway, which incurs additional costs.                                     | Deprecated in October 2023.                  |
+
+<Message type="important">
+  By removing or detaching the Public Gateway from the Private Network, a node pool with full isolation can lead to a single point of failure in the cluster, as nodes will no longer be able to reach their control plane.
+</Message>
+
+## Scaleway product compatibility
+
+### Can I use a Public Gateway with my Private Network to exit all outgoing traffic from the nodes?
+
+Yes, you are required to attach a Private Gateway when setting up a node pool with [full isolation](#what-is-the-difference-between-controlled-isolation-and-full-isolation). This allows Kapsule nodes with private IPs to route their outgoing traffic through the Public Gateway. For detailed steps on setting up a Public Gateway, refer to our [Public Gateway documentation](/network/public-gateways/how-to/configure-a-public-gateway/).
+Keep in mind that removing or detaching the Public Gateway from the Private Network can cause a single point of failure in the cluster, preventing fully isolated node pools from accessing the control plane.
+
+### Is Kosmos compatible with Private Networks?
 
 Only Kapsule can use a Private Network.
+
 Kosmos uses Kilo as a CNI, which uses WireGuard to create a VPN Mesh between nodes for communication between pods. Any node in Kosmos, either in Scaleway or outside, uses these VPN tunnels to communicate securely by construct.
 
-## What future options will there be for isolation?
+### Are Managed Databases compatible with Kubernetes Kapsule on Private Networks?
 
-We are currently at the first phase of an isolated Kubernetes infrastructure. In the future, additional features are planned, such as:
-- compatibility with Scaleway Public Gateways, to be used as a single exit point for the cluster
-- extended VPC integration of Kubernetes Kapsule, with support of Multi-AZ features.
-- Isolation of the control plane on top of your worker nodes
+Yes they are.
 
-## Will the control plane also be located inside the Private Network?
-Currently, only worker nodes are located in the Private Network of your cluster. The communication between the nodes and the control plane uses the Public IP of the node.
+Since July 2023, the automatic allocation of IP addresses via IPAM is available for Managed Databases. These IP addresses are compatible with Scaleway's VPC, which is now in General Availability and can be used with, not only Managed Databases, but all products in the Scaleway ecosystem.
```

Comment:
Not quite, there are some incompatible products (e.g. Messaging and Queuing for one). Maybe link to this page https://www.scaleway.com/en/docs/network/vpc/troubleshooting/vpc-limitations/

Reply:
```suggestion
Since July 2023, the automatic allocation of IP addresses via IPAM is available for Managed Databases. These IP addresses are compatible with Scaleway's VPC, which is now in General Availability. For more information about product compatibility refer to the [VPC documentation]([/network/vpc/troubleshooting/vpc-limitations/](https://www.scaleway.com/en/docs/network/vpc/troubleshooting/vpc-limitations/).
```